jax/nn/__init__.pyi

```diff
@@ -0,0 +1,99 @@
+from typing import Any, List, Literal, Union, overload, Sequence
+
+import jax.numpy as jnp
```

Comment:
We should avoid importing this here, I don't think it's necessary once default values are removed (see below)